docs/Using Price Feeds/get-the-latest-price-solana.md

```diff
@@ -45,7 +45,7 @@ This guide requires the following tools:
   - [Rust](https://www.rust-lang.org/tools/install)
   - [The Solana CLI](https://docs.solana.com/cli/install-solana-cli-tools#use-solanas-install-tool)
   - [Git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git)
-- Call the deployed program to read from the [SOL / USD price data feed](https://explorer.solana.com/address/FmAmfoyPXiA8Vhhe6MZTr3U6rZfEZ1ctEHay1ysqCqcf?cluster=devnet):
+- Call the deployed program to read from the [data feed for SOL / USD prices](https://explorer.solana.com/address/FmAmfoyPXiA8Vhhe6MZTr3U6rZfEZ1ctEHay1ysqCqcf?cluster=devnet):
   - [Node.js 12 or higher](https://nodejs.org/en/download/)
```

Comment:
I think this should not be plural as it's one price - > data feed for SOL / USD price

Reply:
Updated. We can make this part of our style guide for how to refer to data feeds.